unison-src/transcripts/merges.output.md

```diff
@@ -309,7 +312,8 @@ We can also delete the fork if we're done with it. (Don't worry, it's still in t
       builtin.io2.Tls.receive builtin.io2.Tls.send
       builtin.io2.Tls.terminate
       builtin.io2.TlsFailure##TlsFailure
-      builtin.io2.TlsFailure#o6b builtin.todo
+      builtin.io2.TlsFailure#o6b builtin.metadata.isPropagated
+      builtin.metadata.isTest builtin.todo
```

Comment:
This is unchanged since before the PR, but what's going on with these hash-qualified identifiers?  It shows up on some others earlier in this list too, not just TLS stuff.